roles/nginx_configs/meta/main.yml

```diff
@@ -3,5 +3,3 @@ dependencies:
   - { role: nginx }
   - { role: users }
   - { role: passenger }
-  - { role: php }
```

Comment:
Should find a better solution for this. Some hosts don't need PHP, but others who use PHP also use this role.